Types/ModuleInstallDescriptor.cs

```diff
@@ -186,14 +198,15 @@ public ModuleInstallDescriptor ConvertFindToFile(ZipFile zipfile)
             if (candidates.Count == 0)
             {
                 throw new FileNotFoundKraken(
-                    this.find,
-                    String.Format("Could not find {0} directory in zipfile to install", this.find)
+                    this.find ?? this.find_regexp,
```

Comment:
Some of the developers are on mono 3.2.8 which does not include c# 6 features such as ??.